database/src/main/resources/db/V2__Logviewer.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE logevents
```

Comment:
You currently added this to the same database the main application uses. Does the main application need this table and does this application need the users table? It might make more sense to separate them?

Reply:
I saw that the configuration from the Bot-DB was also in the Database project, so I thought it's supposed to be there.

The Bot doesn't need anything from the viewer but I don't know where I would separate them without also breaking the code-generation

Reply:
Yea, that's a bit of an oddity as a few modules share their database apparently (at least that seems to be the intention of whoever extracted the database) but others do not need to. I will have a look at splitting off the viewer db.

Reply:
I have created https://github.com/Together-Java/TJ-Bot/commit/46cb59ed4e3f5d20067fb6d381c9f26c8307a491 for this. If this works for you and you think it's fine, feel free to cherry-pick it into this PR.

Reply:
Work's for me 👍🏽